src/redux/actions/samples/createSample.js

```diff
@@ -83,6 +89,11 @@ const createSample = (
       payload: { sample: newSample, experimentId },
     });
 
+    await dispatch({
+      type: SAMPLES_SAVED,
+      payload: { sample: newSample, experimentId },
+    });
```

Comment:
because I didn't want to refactor redux unless it is the most simple way of implementing the v2 process upload.

right now changing `SAMPLES_SAVED` means also affecting the api v1 action, and I didn't want to get into that.